diff --git a/views_bulk_operations.module b/views_bulk_operations.module
index c5b76d5..b307045 100644
--- a/views_bulk_operations.module
+++ b/views_bulk_operations.module
@@ -764,7 +764,7 @@ function views_bulk_operations_form_submit($form, &$form_state) {
 
   // Redirect.
   $query = drupal_get_query_parameters($_GET, array('q'));
-  $form_state['redirect'] = array('path' => $vbo->view->get_url(), array('query' => $query));
+  $form_state['redirect'] = array($vbo->view->get_url(), array('query' => $query));
 }
 
 /**
